src/libs/Navigation/Navigation.js

```diff
@@ -148,6 +148,70 @@ function setParams(params, routeKey) {
     });
 }
 
+/**
+ * Replace a given reportID in the navigation stack.
+ *
+ * @param {String} oldReportID
+ * @param {String} newReportID
+ */
+function replaceReportIDInNavigationStack(oldReportID, newReportID) {
+    const state = navigationRef.getState();
+
+    const topmostCentralPane = getTopmostCentralPane(state);
+    if (!topmostCentralPane) {
```

Comment:
Would be good to add some comment here about why we don't need to replace anything for this case.